media/src/polls/Polls.svelte

```diff
@@ -76,6 +76,7 @@
     };
 
     const onClickBroadcast = function(e, p) {
+        pollResponses = {};
         currentPollType = p.type;
 
         // Clear current poll responses
```

Comment:
It looks like we had this code in here already. We can remove this comment if this is working now.